api/backend/models.py

```diff
@@ -57,13 +65,15 @@ def __repr__(self):
 
 
 class Tag(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
+    id = db.Column(db.Integer, primary_key=True, nullable=False)
     name = db.Column(db.Text, nullable=False)
     summary = db.Column(db.Text, nullable=False)
     submission_guideline = db.Column(db.Text, nullable=False)
     about = db.Column(db.Text, nullable=False)
     emoji = db.Column(db.Text, nullable=False)
-    users = db. relationship("User", back_populates="tag")
+    users = db.relationship("User", secondary="user_tag_rel",               
```

Comment:
Same as last one, implement the relationship with back_populates instead of backref